repour/info.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from aiohttp import web
+
+logger = logging.getLogger(__name__)
+
+#
+# Info/Documentation endpoint
+#
+
+# @asyncio.coroutine
+class InfoEndpoint:
+    info_data = None
+
+    @staticmethod
+    def handle(request):
+        if InfoEndpoint.info_data is None:
+            InfoEndpoint.info_data = ""
+            with open("README.html", "r") as file:
+                InfoEndpoint.info_data += file.read()
+        return web.Response(body = InfoEndpoint.info_data.encode('utf-8'))
```

Comment:
The encoded (bytes form) of the text should be cached, so it doesn't need to be re-encoded every request. This can either be done by reading the file in text mode (as it is now), and then doing the UTF-8 encoding once, or just read the file in bytes mode (append `b` to the mode string).